pkg/storaged/storage.scss

```diff
@@ -500,6 +500,34 @@ td.storage-action {
     font-size: var(--pf-global--FontSize--xs);
 }
 
+.sidepanel-row:not(.pf-c-empty-state):hover {
```

Comment:
This is basically making the content of the side table rows look like a link. Instead of making it look like one, can we just make it a link and avoid these overrides?

Reply:
Probably. I'd need help from someone to have this happen.

Reply:
Something like https://github.com/cockpit-project/cockpit/commit/adbce4f0a4c017c64de674af349438a7f30eee6b

Reply:
And this one to use hrefs instread of onClick handlers on the links, https://github.com/KKoukiou/cockpit/commit/7fe59a41bbcbe86b3f57ebb92123574df486d883